pkg/logger/common/common.go

```diff
@@ -151,6 +152,14 @@ func fullAccessLogFilter(req *restful.Request, resp *restful.Response, chain *re
 		tokenClientID = jwtClaims.ClientID
 	}
 
+	// mask sensitive field(s) in request body and response body
+	if maskedRequestFields := req.Attribute(log.MaskedRequestFields); maskedRequestFields != nil && requestBody != "" {
```

Comment:
if the request has type `URL-encoded`, would the request body be not empty?

Reply:
is `URL-encoded` means `application/x-www-form-urlencoded` ?
yes, the request body will not empty.
it will catch the data, example: `grant_type=client_credentials`
@anggorodewanto

Reply:
it reads the body from `bodyBytes, err := ioutil.ReadAll(req.Request.Body)` have you double checked that params in URL will be converted into `Request.Body`?

Reply:
No, for query params in URL, we still considered it as URL/Path in Log Viewer:
![image](https://user-images.githubusercontent.com/17809418/152125215-b3cd8c43-25f4-4aba-b7b9-61616eed51b3.png)

But for `application/x-www-form-urlencoded` data, we read it as `requestBody`:
![image](https://user-images.githubusercontent.com/17809418/152124932-46a58119-cb9c-486c-8347-47b3f959a2fd.png)

Reply:
i see, good to know. Will you also filter query param then?

Reply:
yes, will add new option `MaskedQueryParams`

Reply:
updated
@anggorodewanto